docs/tutorials/custom-inference-backend.md

```diff
@@ -1,6 +1,6 @@
 # Custom Inference Backend
 
-This article demonstrates how to add a custom inference backend that is not yet supported by GPUStack, using TensorRT-LLM as an example.
+This article demonstrates how to add a custom inference backend that is not yet supported by GPUStack, using TensorRT-LLM as an example, at the end of the article, configuration methods for common inference backends are also listed.
```

Comment:
Can we have a `Inference Backend Management` in user-guides, move current `Inference Backends` to `Built-in Inference Backends`, and keep all examples in tutorials?